templates/settings.html

```
@@ -214,6 +217,47 @@ <h5 class="mb-0">
                     </div>
                 </div>
             </div>
+
+            <!-- File Encryption Key Migration Card -->
+            <div id="file-encryption" class="card mb-4">
+                <div class="card-header d-flex justify-content-between align-items-center">
+                    <h5 class="mb-0">
+                        <i class="bi bi-file-earmark-lock me-2"></i>Cold Storage File Encryption
+                    </h5>
+                    <button class="btn btn-sm btn-primary" id="btn-rotate-file-key">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)